src/worker/tasks.ts

```diff
@@ -0,0 +1,51 @@
+import { PluginEvent } from '@posthog/plugin-scaffold/src/types'
+
+import { EnqueuedJob, PluginsServer, PluginTaskType } from '../types'
+import { ingestEvent } from './ingestion/ingest-event'
+import { runPlugins, runPluginsOnBatch, runPluginTask } from './plugins/run'
+import { loadSchedule, setupPlugins } from './plugins/setup'
+import { teardownPlugins } from './plugins/teardown'
+
+type TaskRunner = (server: PluginsServer, args: any) => Promise<any> | any
+
+export const workerTasks: Record<string, TaskRunner> = {
+    hello: (server, args) => {
+        return `hello ${args}!`
+    },
```

Comment:
Should this be in here as just `hello`? Seems a bit cryptic/odd

Reply:
That can be most likely deleted. I just copied this from the previous place. Next time, as there are more refactors to come.